### DISCUSSION TITLE
Welcome to MonitorControl Discussions!

### DISCUSSION THREAD

**Discussion by the0neyouseek** [MonitorControl/MonitorControl#343] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by razvanmicorici** [comment]
Hello there! First of all, let me say how awesome MonitorControl is. It really saves me a lot of time.
I was thinking about two things, and decided to ask you guys if there are some workarounds for them.

First, I was wondering if there is a way to invert the fact that I have to press opt+shift+volume up to change the volume in increments of 1%. I would like to use the regular volume up and down keys and change the volume in increments of 1%. Can I do something to change this?

And secondly, I use a MacBook Pro 16 with and LG34WK95U and when I change the volume, the monitor shows his volume bar as well as the Mac. In the past I was using Soundflower and it only showed me the volume bar from the macos. I would like to know if there is a way to hide the volume bar displayed by the LG monitor.

Thanks!

**Comment by fender357** [comment]
Wonderful app. I'm wondering if you may look to expand its capability around selecting sources on a monitor. In my normal workflow I use a secondary monitor which connects to two different computers. It would be great to be able to switch the sources of that monitor from my Mac.
